### ISSUE TITLE
no longer able to copy and paste coordinates into wxMacMolPlt

### ISSUE THREAD

**Issue by xupengpeng02** [brettbode/wxmacmolplt#145]
### Please describe the problem or enhancement:

I used to be able to copy and paste the Cartesian coordinates from GAMESS input or output files directly into a wxMacMolPlt window to visualize the structure. 
After I download the latest version (v7.7.2) to my new MacBook Pro (Apple M2 Pro, Ventura 13.5.1) I can no longer do that.

### If this is a bug report please list the steps required to reproduce the problem:

open a new window in wxMacMolPlt
try copying and pasting the following coordinates for CO2 molecule

 C           6.0  -0.0000000000   0.0000000000  -0.0000000001
 O           8.0  -0.0000000000  -1.1695476346   0.0000000001
 O           8.0  -0.0000000000   1.1695476346   0.0000000001

### What version of wxMacMolPlt are you running?
v7.7.2
### What OS are you using? (please include 32/64 bit and if Linux the specific distribution)
Apple M2 Pro, macOS Ventura 13.5.1

**Comment by brettbode** [comment]
I have confirmed this issue and unfortunately found it persists in the latest code. It appears to me that the problem is in the wxWidgets library that MacMolPlt is built upon. That makes fixing it a bit more challenging though.

**Comment by brettbode** [comment]
After debugging this a bit more I determined that the issue is a change in behavior in some combination of recent MacOS versions or wxWidgets version where incoming text seems to be classified as unicode text even when coming from plain text documents. I have modified the code for enabling paste and handling the text to allow pasting limited unicode text. The change is checked in and will be in the next release.
